pkg/plugin/extension_test.go

```diff
@@ -5,27 +5,30 @@ import (
 
 	"github.com/reearth/reearth-backend/pkg/i18n"
 	"github.com/reearth/reearth-backend/pkg/id"
+	"github.com/reearth/reearth-backend/pkg/scene"
 	"github.com/reearth/reearth-backend/pkg/visualizer"
 	"github.com/stretchr/testify/assert"
 )
 
 func TestExtension(t *testing.T) {
 	expected := struct {
-		Id          id.PluginExtensionID
-		Type        ExtensionType
-		Name        i18n.String
-		Description i18n.String
-		Icon        string
-		Schema      id.PropertySchemaID
-		Visualizer  visualizer.Visualizer
+		Id           id.PluginExtensionID
+		Type         ExtensionType
+		Name         i18n.String
+		Description  i18n.String
+		Icon         string
+		Schema       id.PropertySchemaID
+		Visualizer   visualizer.Visualizer
+		WidgetLayout *scene.WidgetLayout
 	}{
-		Id:          "xxx",
-		Type:        ExtensionTypePrimitive,
-		Name:        i18n.StringFrom("aaa"),
-		Description: i18n.StringFrom("ddd"),
-		Icon:        "test",
-		Schema:      id.MustPropertySchemaID("hoge#0.1.0/fff"),
-		Visualizer:  "vvv",
+		Id:           "xxx",
+		Type:         ExtensionTypePrimitive,
+		Name:         i18n.StringFrom("aaa"),
+		Description:  i18n.StringFrom("ddd"),
+		Icon:         "test",
+		Schema:       id.MustPropertySchemaID("hoge#0.1.0/fff"),
+		Visualizer:   "vvv",
+		WidgetLayout: &scene.WidgetLayout{},
```

Comment:
As I mentioned in another comment, don't empty it, fill it with something inside.